src/components/UniversalTable/UniversalTable.tsx

```diff
@@ -0,0 +1,30 @@
+import {  ReactNode } from 'react';
+import classes from './UniversalTable.module.scss';
+
+type UniversalTableProps<D> = {
+  data: D[]
+
+  headers: {
+    title: string;
+    property: keyof D;
+  }[]
+}
+
+const UniversalTable = <T,>({data, headers}:UniversalTableProps<T>):ReactNode => {
+  return (
+    <>
+      <div className={classes.TableContainer}>
+        <ul className={classes.TableHeader}>
+          {headers.map((header)=> <li key={header.title} className={classes.TableHeaderCell}>{header.title}</li>)}
+        </ul>
+        {data.map((row, rowIndex) => (
+          <ul key={rowIndex} className={classes.TableRow}>
+            {headers.map((header, cellIndex)=> <li key={cellIndex} className ={classes.TableRowCell}>{row[header.property]}</li>)}
```

Comment:
![image](https://github.com/BUI-studies/yidlorama/assets/109099364/a3345b55-a90a-4e90-901b-2274dd5b09a8)

Reply:
![image](https://github.com/BUI-studies/yidlorama/assets/109099364/0e60ce0b-ac29-48ab-bb49-a126d0f00309)

Reply:
1) error disappear
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/5d395ae0-7049-4e78-b082-f10d4e088170)

2) a`ve been commented it to you earlier